viewer/core/src/public/migration/types.ts

```diff
@@ -62,6 +62,20 @@ export interface Cognite3DViewerOptions {
    */
   renderTargetOptions?: { target: THREE.WebGLRenderTarget; autoSetSize?: boolean };
 
+  /**
+   * Style the loading indicator.
+   */
+  loadingIndicatorStyle?: {
+    /**
+     * What corner the spinner should be placed in. Defaults top topLeft.
+     */
+    placement: 'topLeft' | 'topRight' | 'bottomLeft' | 'bottomRight';
```

Comment:
What is the benefit of this vs. an enum?

Reply:
This matches the options in Cognite3DViewer, i.e. how we've done this for antialiasing and ssao